### PR TITLE
[ros] Bump ROS Dashing to 0.7.2

### PR DESCRIPTION
**Pull request by mikaelarguedas**
Bump ros dashing 0.7.0-1 -> 0.7.2-1, this should fix the `ros:dashing-*` builds.

### CODE REVIEW

**Comment by mikaelarguedas** [comment]
travis failing due to the presence of some ubuntu:trusty based images: https://github.com/osrf/docker_images/issues/282

File: library/ros [hunk 1]
```diff
@@ -197,10 +197,10 @@ Directory: ros/crystal/ubuntu/bionic/ros-base
...
 Tags: dashing-ros-core, dashing-ros-core-bionic
 Architectures: amd64, arm64v8
-GitCommit: b832b00758b1cde2893b80f2f769b076164d9982
+GitCommit: 32244a8fafcf7d3c69c48e3a3aa082e4acc1fbf7
 Directory: ros/dashing/ubuntu/bionic/ros-core
 
 Tags: dashing-ros-base, dashing-ros-base-bionic, dashing
 Architectures: amd64, arm64v8
-GitCommit: b832b00758b1cde2893b80f2f769b076164d9982
+GitCommit: 32244a8fafcf7d3c69c48e3a3aa082e4acc1fbf7
```
Comment (yosifkit) [inline library/ros]: Shouldn't these commits be [`bf5aa6ca8ca41a096dd20b7b9e4e18594795bd27`](https://github.com/osrf/docker_images/commit/bf5aa6ca8ca41a096dd20b7b9e4e18594795bd27)?

Comment (mikaelarguedas) [reply library/ros]: Indeed, something must have gone wrong in the library file generation, regenerated in https://github.com/docker-library/official-images/pull/6132/commits/d37d6b8190e5105705da26299a21eaa385a0a559
Thanks for catching that!

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/ros_dashing-ros-core-bionic/Dockerfile b/ros_dashing-ros-core-bionic/Dockerfile
index fc9f01a..7035708 100644
--- a/ros_dashing-ros-core-bionic/Dockerfile
+++ b/ros_dashing-ros-core-bionic/Dockerfile
@@ -44,7 +44,7 @@ RUN pip3 install -U \
 # install ros2 packages
 ENV ROS_DISTRO dashing
 RUN apt-get update && apt-get install -y \
-    ros-dashing-ros-core=0.7.0-1* \
+    ros-dashing-ros-core=0.7.2-1* \
     && rm -rf /var/lib/apt/lists/*
 
 # setup entrypoint
diff --git a/ros_dashing/Dockerfile b/ros_dashing/Dockerfile
index a3a7ead..5f67e5b 100644
--- a/ros_dashing/Dockerfile
+++ b/ros_dashing/Dockerfile
@@ -3,6 +3,6 @@
 FROM ros:dashing-ros-core-bionic
 # install ros2 packages
 RUN apt-get update && apt-get install -y \
-    ros-dashing-ros-base=0.7.0-1* \
+    ros-dashing-ros-base=0.7.2-1* \
     && rm -rf /var/lib/apt/lists/*
 
```

</details>

**Comment by yosifkit** [comment]
Build test of #6132; d37d6b81; `amd64` (`ros`):

```console

$ bashbrew build ros:indigo-ros-core
Using bashbrew/cache:c925183fcb05765af2e2bdd8686efaf06b9fb55d583b6de352ef34b477cdc863 (ros:indigo-ros-core)
Tagging ros:indigo-ros-core
Tagging ros:indigo-ros-core-trusty

$ test/run.sh ros:indigo-ros-core
testing ros:indigo-ros-core
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:indigo-ros-base
Using bashbrew/cache:375079a4d307257ab70a16b7e5118a5cdaaaa4cddb686b03d486484e00812818 (ros:indigo-ros-base)
Tagging ros:indigo-ros-base
Tagging ros:indigo-ros-base-trusty
Tagging ros:indigo

$ test/run.sh ros:indigo-ros-base
testing ros:indigo-ros-base
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:indigo-robot
Using bashbrew/cache:59600ca9666e67dbe0643503f1df8fb31ec2c39293cd4c6511d25fdd8dae56fc (ros:indigo-robot)
Tagging ros:indigo-robot
Tagging ros:indigo-robot-trusty

$ test/run.sh ros:indigo-robot
testing ros:indigo-robot
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:indigo-perception
Using bashbrew/cache:1d1ce9f5ff414e2d0ef9cb7cc0ade7f983bc506096418f24c1cd159ddc8ed24d (ros:indigo-perception)
Tagging ros:indigo-perception
Tagging ros:indigo-perception-trusty

$ test/run.sh ros:indigo-perception
testing ros:indigo-perception
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:kinetic-ros-core
Using bashbrew/cache:f712f6684f1904a6eef7015d046312bcaa832dec24db89ac3a2ceb6e2fb3b251 (ros:kinetic-ros-core)
Tagging ros:kinetic-ros-core
Tagging ros:kinetic-ros-core-xenial

$ test/run.sh ros:kinetic-ros-core
testing ros:kinetic-ros-core
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:kinetic-ros-base
Using bashbrew/cache:99b5a1b731c0e4fdab3399d077dc4caa833b9850d657da1e4291408027378163 (ros:kinetic-ros-base)
Tagging ros:kinetic-ros-base
Tagging ros:kinetic-ros-base-xenial
Tagging ros:kinetic

$ test/run.sh ros:kinetic-ros-base
testing ros:kinetic-ros-base
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:kinetic-robot
Using bashbrew/cache:f177a8c232186b848f86aafadc3652ed1305bea266f7db88cc7ca751a5e337b2 (ros:kinetic-robot)
Tagging ros:kinetic-robot
Tagging ros:kinetic-robot-xenial

$ test/run.sh ros:kinetic-robot
testing ros:kinetic-robot
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:kinetic-perception
Using bashbrew/cache:8b112a7bfd3d219c65590c134bef8e500673be5cdf3054f11f300b17e2f2895b (ros:kinetic-perception)
Tagging ros:kinetic-perception
Tagging ros:kinetic-perception-xenial

$ test/run.sh ros:kinetic-perception
testing ros:kinetic-perception
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:lunar-ros-core
Using bashbrew/cache:8963ed7fcea7934287f46fa10bc0984d8c84a20b77e6f9cb395b5ce241fbb46a (ros:lunar-ros-core)
Tagging ros:lunar-ros-core
Tagging ros:lunar-ros-core-xenial

$ test/run.sh ros:lunar-ros-core
testing ros:lunar-ros-core
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:lunar-ros-base
Using bashbrew/cache:8d53bdbcb295aeebfd182aba5784cb2c830a5eac59695ea2e83913257a9e2376 (ros:lunar-ros-base)
Tagging ros:lunar-ros-base
Tagging ros:lunar-ros-base-xenial
Tagging ros:lunar

$ test/run.sh ros:lunar-ros-base
testing ros:lunar-ros-base
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:lunar-robot
Using bashbrew/cache:2ef5577ce219e828ee3f7910e2d69e634348bf158067b34033dc317df3fd7074 (ros:lunar-robot)
Tagging ros:lunar-robot
Tagging ros:lunar-robot-xenial

$ test/run.sh ros:lunar-robot
testing ros:lunar-robot
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:lunar-perception
Using bashbrew/cache:6d7d6a0fa9ff65c72a59e4a0d1d1a2cc1811aedb9d4bc5e91710a61e91326f62 (ros:lunar-perception)
Tagging ros:lunar-perception
Tagging ros:lunar-perception-xenial

$ test/run.sh ros:lunar-perception
testing ros:lunar-perception
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:lunar-ros-core-stretch
Using bashbrew/cache:614e9a80d8a8105a88307690aba465dcc2224046168691d46637e8b39db8f3d8 (ros:lunar-ros-core-stretch)
Tagging ros:lunar-ros-core-stretch

$ test/run.sh ros:lunar-ros-core-stretch
testing ros:lunar-ros-core-stretch
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:lunar-ros-base-stretch
Using bashbrew/cache:5c8c1f95a1c00c7ddad70942024a6a451509a93010cfb2f44ad3a92ef9f48ea7 (ros:lunar-ros-base-stretch)
Tagging ros:lunar-ros-base-stretch

$ test/run.sh ros:lunar-ros-base-stretch
testing ros:lunar-ros-base-stretch
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:lunar-robot-stretch
Using bashbrew/cache:e65d0743544ff0c85e76fd0c5d034a2efcbc9cfedea544664155cedb6e193914 (ros:lunar-robot-stretch)
Tagging ros:lunar-robot-stretch

$ test/run.sh ros:lunar-robot-stretch
testing ros:lunar-robot-stretch
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:lunar-perception-stretch
Using bashbrew/cache:5b5a235486f322beb196b70d52710ef4b4b8be46de8704c7dfc1c9e945f76089 (ros:lunar-perception-stretch)
Tagging ros:lunar-perception-stretch

$ test/run.sh ros:lunar-perception-stretch
testing ros:lunar-perception-stretch
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:melodic-ros-core
Using bashbrew/cache:22be55477373bd7c518a54cb24d74c05c2a9e2349d963bcc3e95a4e327396a85 (ros:melodic-ros-core)
Tagging ros:melodic-ros-core
Tagging ros:melodic-ros-core-bionic

$ test/run.sh ros:melodic-ros-core
testing ros:melodic-ros-core
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:melodic-ros-base
Using bashbrew/cache:c64ec6b59933c46e8c0537fc33020b95b48dbbcfbc866687d62b275ad81aaf3e (ros:melodic-ros-base)
Tagging ros:melodic-ros-base
Tagging ros:melodic-ros-base-bionic
Tagging ros:melodic
Tagging ros:latest

$ test/run.sh ros:melodic-ros-base
testing ros:melodic-ros-base
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:melodic-robot
Using bashbrew/cache:bec799965c045cb401245c340732e8df74791b20c42685568135f1ba710745b0 (ros:melodic-robot)
Tagging ros:melodic-robot
Tagging ros:melodic-robot-bionic

$ test/run.sh ros:melodic-robot
testing ros:melodic-robot
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:melodic-perception
Using bashbrew/cache:41dbc476e75699fa9f221968ccfeb24bb38584b2c2f4743eae6d7346d7c8235a (ros:melodic-perception)
Tagging ros:melodic-perception
Tagging ros:melodic-perception-bionic

$ test/run.sh ros:melodic-perception
testing ros:melodic-perception
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:melodic-ros-core-stretch
Using bashbrew/cache:06248416d9f5eb92030e6baea3925caea2b0e1051d7573501c2afc7766ca6cbc (ros:melodic-ros-core-stretch)
Tagging ros:melodic-ros-core-stretch

$ test/run.sh ros:melodic-ros-core-stretch
testing ros:melodic-ros-core-stretch
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:melodic-ros-base-stretch
Using bashbrew/cache:eb7f169e0c64e01fcee4a1acbb913595f69d872e7fef27a5d93e6c1d9a2298b0 (ros:melodic-ros-base-stretch)
Tagging ros:melodic-ros-base-stretch

$ test/run.sh ros:melodic-ros-base-stretch
testing ros:melodic-ros-base-stretch
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:melodic-robot-stretch
Using bashbrew/cache:c39e67670c0f8bd6c29c34bca4807c250f3b13103688516336d63586a1a5fbd1 (ros:melodic-robot-stretch)
Tagging ros:melodic-robot-stretch

$ test/run.sh ros:melodic-robot-stretch
testing ros:melodic-robot-stretch
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:melodic-perception-stretch
Using bashbrew/cache:42773ad24ebfa305f8e253424a9bce21897e35df2c60e07256e9452611540d47 (ros:melodic-perception-stretch)
Tagging ros:melodic-perception-stretch

$ test/run.sh ros:melodic-perception-stretch
testing ros:melodic-perception-stretch
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:bouncy-ros-core
Building bashbrew/cache:75a7309c6841d0ce4787871187e7da7a7f35e24e6dd1c00a99c6a279b23e7738 (ros:bouncy-ros-core)
Tagging ros:bouncy-ros-core
Tagging ros:bouncy-ros-core-bionic

$ test/run.sh ros:bouncy-ros-core
testing ros:bouncy-ros-core
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:bouncy-ros-base
Building bashbrew/cache:0ea4d6e0be3d200d3aae0764d25b493919dea0bccca7b66007766a28eddbff72 (ros:bouncy-ros-base)
Tagging ros:bouncy-ros-base
Tagging ros:bouncy-ros-base-bionic
Tagging ros:bouncy

$ test/run.sh ros:bouncy-ros-base
testing ros:bouncy-ros-base
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:crystal-ros-core
Building bashbrew/cache:f7f9e20628b86a2ace237721f41d9aaae29962cd7ad134d9de20a4d4c3e4a6c3 (ros:crystal-ros-core)
Tagging ros:crystal-ros-core
Tagging ros:crystal-ros-core-bionic

$ test/run.sh ros:crystal-ros-core
testing ros:crystal-ros-core
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:crystal-ros-base
Building bashbrew/cache:fa7945e2080bba5d63edd29e80d7d65434dd4acd4675180909ea88793b4e99e9 (ros:crystal-ros-base)
Tagging ros:crystal-ros-base
Tagging ros:crystal-ros-base-bionic
Tagging ros:crystal

$ test/run.sh ros:crystal-ros-base
testing ros:crystal-ros-base
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:dashing-ros-core
Building bashbrew/cache:5ce679118ddc61237c7f17ea3b7b341d8f4bbdf9d3c958b4d7d74a4fd9941d1e (ros:dashing-ros-core)
Tagging ros:dashing-ros-core
Tagging ros:dashing-ros-core-bionic

$ test/run.sh ros:dashing-ros-core
testing ros:dashing-ros-core
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build ros:dashing-ros-base
Building bashbrew/cache:58cc4d2aff30d3b23358eea9b48c42ae2d78af0f2226d06773045754f9f64f72 (ros:dashing-ros-base)
Tagging ros:dashing-ros-base
Tagging ros:dashing-ros-base-bionic
Tagging ros:dashing

$ test/run.sh ros:dashing-ros-base
testing ros:dashing-ros-base
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**